### DISCUSSION TITLE
Double entries in amperfy

### DISCUSSION THREAD

**Discussion by Khymick** [BLeeEZ/amperfy#184] [Q&A]
Hi there,

First of all: great work so far. Amperfy is the only client that is able to connect to my Ampache server. 

The only problem I am facing: when I change some id3 tags of the files, Amperfy lists all albums of on artist two times. That said: before the changes, I had 35 albums listed when I select a specific artist. I noticed that there were errors in the id3 tags and corrected them. Now, Amperfy detects 70 albums of that artist. 

Additionally, I can say that:
- all other subsonic clients I tried so far give me “could not connect to subsonic server” 
- In the web interface, the number of albums for the artist mentioned above is correctly displayed as 35
- Below the 70 albums in amperfy, I see only 35 albums (that means that this is correct), but all of them are marked as “verwaist” (sorry, I use German interface). 

This looks like the database is not correctly re-read when I change something on the files server-sided. Also, on the Ampache side, all is correct. So it seems that this may be a problem of the client. 

Can you help me solve the issue? If you need further infos, please let me know.

**Comment by BLeeEZ** [comment]
Hi,
thank you very much. Theses kind of library updates are very difficult to merge with the Amperfy library database. You can try out the current beta v1.1.3 (1) https://github.com/BLeeEZ/amperfy/issues/25. In this version you can resync the library which should definitely fix this issue. You can trigger the resync in beta via: Settings->Library->"Resync Library".

**Reply by Khymick** [reply]
Hi,

thanks for your reply. I did install the current beta and I will try if this solves the problem. 

Generally, let me add a question: if I have new folders on my server (not present before), how can I get Amperfy to scan for new music?

**Comment by BLeeEZ** [comment]
Amperfy does this in the background. You can also trigger it by visiting „Recent Songs“ or „Recent Albums“.

**Reply by Khymick** [reply]
Thanks for reply. Just to make sure: when I click on “recent albums”, the scan for new albums is automatically triggered?

i also noticed that when I scroll down in the main screen, some refresh is triggered. Is this the same refresh?

**Comment by cdwieber** [comment]
Hate to necro a 3+ year old thread, but should someone stumble upon this via Googling like I have, I think I found a fix: On the Ampache server, I went to the catalog and ran "Garbage Collect" and "Clean" in that order, then resync'd the library in Amperfy as above (it's under "Settings" -> "Accounts" now). Now there are no more empty phantom duplicates in my Amperfy library.
